tests/conftest.py

```diff
@@ -41,3 +45,29 @@ def cdn_test_url(request):
     else:
         pytest.skip("Test skipped without --cdn-test-url or --lambda-stack")
     return url
+
+
+def mock_conf_file():
+    temp_file = tempfile.NamedTemporaryFile(delete=False)
```

Comment:
Also please use a 'prefix' or 'suffix'.  It's a good practice so that if /tmp fills up with files, you can get some idea of which code created those files.

Reply:
```
temp_file = tempfile.NamedTemporaryFile(prefix="lambda_unitests_", delete=False)
```
will create tempfile, like "/tmp/lambda_unitests_jypgdwwz", seems OK.